wallet/wallet.go

```diff
@@ -450,6 +451,20 @@ func (w *Wallet) syncWithChain(birthdayStamp *waddrmgr.BlockStamp) error {
 	return w.rescanWithTarget(addrs, unspent, nil)
 }
 
+// isDevEnv determines whether the wallet is currently under a local developer
+// environment, e.g. simnet or regtest.
+func (w *Wallet) isDevEnv() bool {
+	switch uint32(w.ChainParams().Net) {
+	case uint32(chaincfg.RegressionNetParams.Net):
+	case uint32(chaincfg.SimNetParams.Net):
+	case uint32(ltcChainCfg.RegressionNetParams.Net):
```

Comment:
Nothing in btcwallet is aware of LTC, we shouldn't introduce awareness here.

Reply:
Fixed.